docs/discover/search.asciidoc

```diff
@@ -88,6 +88,81 @@ If the saved search is associated with a different index pattern than is current
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving queries
+A saved query is a portable collection of query text and filters that you can reuse in <<discover, Discover>>, <<visualize, Visualize>> and <<dashboard, Dashboard>>. Save a query when you want to:
+
+* Retrieve results from the same query at a later time without having to reenter the query text, add the filters or set the time filter again
+* Share your query
+* View the results of the same query in multiple apps
+
+Saved queries don't include information specific to Discover, such as the currently selected columns in the document table, the sort order, and the index pattern. If you want to save your current view of Discover for later retrieval and reuse, create a <<save-open-search, saved search>> instead.
+
+[role="xpack"]
+==== Read only access
+If you have insufficient privileges to save queries, the *Save current query* button isn't visible in the saved query management popover.
+
+[role="screenshot"]
+image::discover/images/saved-query-management-component-readonly-view.png["Example of the saved query management popover with read only access to Discover",width="80%"]
+
+==== Saving a query
+To save the current query text, filters, and time filter:
+
+. Click *#* in the search bar, next to the query text input.
+. Click *Save current query* in the popover.
++
+[role="screenshot"]
+image::discover/images/saved-query-management-component-all-privileges.png["Example of the saved query management popover with a list of saved queries with write access",width="80%"]
++
+. Enter a name, a description, and then select the filter options that you want to include. By default filters are automatically included, but the time filter is not.
++
+[role="screenshot"]
+image::discover/images/saved-query-save-form-default-filters.png["Example of the saved query management save form with the filters option included and the time filter option excluded",width="80%"]
+. Click *Save*.
+
+==== Loading a query
+To load a saved query into Discover, Dashboard, or Visualize:
+
+. Click *#* in the search bar, next to the query text input.
+. Select the query you want to load. You might need to scroll down to find the query you are looking for.
+
+==== Saving changes to a query
+If you load a query and then make changes to the query text, the filters, or the time filter, you can save the changes as a new query or update the existing query.
+
+To save the changes as a new query:
+
+. Click *#* in the search bar, next to the query text input.
+. Click *Save as new* in the popover.
+. Enter a name and a description, and then select the filter options that you want to include.
+. Click *Save*.
++
+[role="screenshot"]
+image::discover/images/saved-query-management-component-save-as-new-query.png["Example of the saved query management popover when a query is loaded and we have made changes to the query",width="80%"]
```

Comment:
I don't think this image is necessary.

Reply:
We need it to show the 'Clear', 'Save as new' and 'Save changes' buttons. Those buttons aren't in the other screen shots.